test/quantization/test_tensorflow_qat.py

```diff
@@ -101,13 +101,10 @@ def q_func(model):
     train_images = train_images / 255.0
     test_images = test_images / 255.0
 
-    model = tf.keras.models.load_model("baseline_model")
-
-    import tensorflow_model_optimization as tfmot
-    quantize_model = tfmot.quantization.keras.quantize_model
-
-    # q_aware stands for for quantization aware.
-    q_aware_model = quantize_model(model)
+    # apply qat convert
+    from neural_compressor.experimental import ModelConversion
+    conversion = ModelConversion()
```

Comment:
shouldn't use model conversion to do convert.

Reply:
Will solve the current issues after completing examples modification of newAPI.